packages/account-v2/src/context/SignupWizardContext.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { createContext, useContext, useMemo, useState } from 'react';
+import { useStep } from 'usehooks-ts';
+
+type Helpers = ReturnType<typeof useStep>[1];
+
+type TSignupWizardContext = {
+    currentStep: number;
+    helpers: Helpers;
+    isWizardOpen: boolean;
+    setIsWizardOpen: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+type TSignupWizardProvider = {
+    children: React.ReactNode;
+};
+
+const initialHelpers: Helpers = {
+    canGoToNextStep: false,
+    canGoToPrevStep: false,
+    goToNextStep: /* noop */ () => {
```

Comment:
What does noop mean?

Reply:
This is to identity an empty function as a initial value